spec/features/invoices/update_spec.rb

```diff
@@ -40,16 +40,12 @@
       within "##{dom_id(invoice_with_items)}" do
         click_on 'Edit'
       end
-      page.execute_script("$(\"##{dom_id(invoice_with_items, :edit)} #invoice_amount\").val('');")
-      fill_in 'invoice[amount]', with: new_amount
       first('#invoice .nested-fields input.nested-amount').set(new_item_amount)
       first('#invoice .nested-fields textarea.nested-description').set('First Nested Description')
       click_on 'Update Invoice'
     end
 
     it { expect(page).to have_content 'Invoice was successfully updated' }
-    it { expect(page).to have_css("##{dom_id(invoice_with_items)} td",
-      text: money_with_symbol(Money.new(new_amount, invoice.currency))) }
```

Comment:
@yunixon add checking that amount is disabled right after opening Edit page, and then after adding/removing InvoiceItems.